### DISCUSSION TITLE
V2 API "fup" & "ids"

### DISCUSSION THREAD

**Discussion by BenjaminK87** [MK-2001/ioBroker.go-e#229] [Ideas]
Guten Morgen,

wäre es möglich die beiden Keys fup und ids aus der V2 API zur Verfügung zu stellen? 
Dann könnte man die interne Solarüberschuss Funktion von der App/Charger nutzen.

Grüße

**Comment by Dax88** [comment]
Das wär wirklich super!

**Reply by BenjaminK87** [reply]
Ich denke in der neuesten Github Version ist es enthalten:
#240
